### PR TITLE
Removed memory tab

### PR DESCRIPTION
**Pull request by dg**

### CODE REVIEW

**Comment by hrach** [comment]
:+1: I understand the icon as a temporary, so I don't care.

**Comment by vysinsky** [comment]
:+1:

File: src/Tracy/templates/bar.statistics.panel.phtml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	<td>Execution time</td>
+	<td><b><?php echo number_format((microtime(TRUE) - Debugger::$time) * 1000, 1, '.', ' ') ?></b> ms</td>
+</tr>
+<tr>
+	<td>The peak of allocated memory</td>
+	<td><b><?php echo number_format(memory_get_peak_usage() / 1000000, 2, '.', ' ') ?></b> MB</td>
+</tr>
+<tr>
+	<td>Included files</td>
+	<td><b><?php echo count(get_included_files()), '</b> (', number_format(array_sum(array_map('filesize', get_included_files())) / 1000000, 2, '.', ' ') ?> MB)</td>
```
Comment (fprochazka) [inline src/Tracy/templates/bar.statistics.panel.phtml]: i don't think the filesize is neccesary

Comment (dg) [inline src/Tracy/templates/bar.statistics.panel.phtml]: Only execution time is neccesary.

**Comment by enumag** [comment]
:+1:

**Comment by JanTvrdik** [comment]
How is IP address and PHP version related to statistics? Why not make Tracy logo clickable instead and write environment info there instead of the title attribute?

**Comment by dg** [comment]
Renamed to 'info'

**Comment by JanTvrdik** [comment]
Will you simplify the Tracy logo title?

**Comment by hrach** [comment]
I don't see added value in number of classes etc, at least strip funcs. It's hard to read the rest.

**Comment by fprochazka** [comment]
`Types + functions: 344 + 2` would be nicer imho

**Comment by Majkl578** [comment]
@fprochazka: Trait is not a type.

**Pull request merged**